ingest-app/docs/api.md

```diff
@@ -1822,16 +1803,149 @@ curl -XPOST \
     "name": "Update OnlineResource URL values",
     "operation": "UPDATE_FIELD",
     "update-field": "OnlineResourceURL",
-    "updates": [
-        {
-            "GranuleUR": "my-gran-ur",
-            "Links": [
-                { "from": "https://old-link", "to": "http://new-link" }
-            ]
-        }
-    ]
+    "updates": [{"GranuleUR": "my-gran-ur",
+                 "Links": [{ "from": "https://old-link", "to": "http://new-link" }]}]
+}'
```

Comment:
From the examples given, seems that just use {"URL": "http://update-to-new-link.com"} is cleaner, than "from" "to".

Reply:
I need to have the from and to.  The from lets the code know which of the related urls, online resources, online access urls, or image urls I need to replace out of the many that can exist.  The to: the code needs to know what to replace it with.